tasks/grunt-aws-cloudformation.js

```diff
@@ -109,6 +109,16 @@ module.exports = function(grunt) {
 					}
 				});
 				break;
+			case "stack-status":
+				var statusParams = {
+					StackName: data.stackName,
+					NextToken: data.nextToken
+				};
+				cloudformation.describeStacks(statusParams, function(err, data) {
+					if (err) grunt.warn(err, err.stack);
+					else     grunt.log.writeln("Stack status:\n" + JSON.stringify(data, null, 1));
```

Comment:
Oh ofc, I havent written much for grunt in the past.
I'll probably add some more functions to this as my needs increase :)